intellij/src/saros/intellij/ui/wizards/pages/referencepointselection/ReferencePointTab.java

```diff
@@ -580,21 +579,24 @@ private void updateInputValidity() {
    */
   // TODO check whether resource with same name already exists in chosen base directory
   private boolean hasValidNewDirectoryName() {
-    String enteredName = newDirectoryNameTextField.getText();
-    if (enteredName.isEmpty()) {
+    String enteredName = newDirectoryNameTextField.getText().trim();
+
+    if (enteredName.isEmpty()
+        || enteredName.contains(File.separator)
+        || enteredName.contains(File.pathSeparator)
```

Comment:
Have you tested what will happen on Windows systems if you use `/` ?

Reply:
I knew there was a reason why the initial solution I had stashed away used the path utility to check the name count. Yes, Windows will also accept '/' as separator character, even though it is not specified as the Java File.separator character.

I will modify the logic. I wanted to explicitly check for the separator to also disallow including it in the name without adding a new path segment (e.g. giving `foo/` instead of `foo`). But I guess it doesn't matter that much.

Reply:
Done. Moved to using the Path API to check for the name count and whether the path is absolute.